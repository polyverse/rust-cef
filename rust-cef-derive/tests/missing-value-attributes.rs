#[macro_use]
extern crate rust_cef_derive;

#[derive(
    CefHeaderVersion,
    CefHeaderDeviceVendor,
    CefHeaderDeviceProduct,
    CefHeaderDeviceVersion,
    CefHeaderDeviceEventClassID,
    CefHeaderName,
    CefHeaderSeverity,
    CefExtensions,

)]
struct MissingValueAttributes {}

fn main() {

}