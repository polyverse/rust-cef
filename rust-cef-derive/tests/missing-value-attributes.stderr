error: When deriving trait CefHeaderVersion, no attribute was found to define a fixed value for it (i.e. #[cef_header_version_fixed(value)])
  --> $DIR/missing-value-attributes.rs:15:1
   |
15 | struct MissingValueAttributes {}
   | ^^^^^^
